docs/docsite/rst/dev_guide/developing_modules_checklist.rst

```diff
@@ -46,7 +46,7 @@ The complete module metadata specification is here: https://github.com/ansible/p
     * For password / secret arguments no_log=True should be set.
     * Requirements should be documented, using the `requirements=[]` field.
     * Author should be set, with their name and their github id, at the least.
-    * Ensure that you make use of U() for urls, C() for files and options, I() for params, M() for modules.
+    * Ensure that you make use of `U()` for URLs, `I()` for option names, `C()` for files and option values, `M()` for module names.
```

Comment:
Using option instead of parameter is more in accordance with the `DOCUMENTATION` block where all **options** are under the `options:` map key.